public/controllers/brandController.js

```diff
@@ -113,7 +155,10 @@ const deleteBrand = async (req, res) => {
   );
 
   if (getAllProductsError) {
-    log.error('ERR_BRAND_GET-ALL-PRODUCTS', error.message);
+    log.error('ERR_BRAND_GET-ALL-PRODUCTS', {
+      err: getAllProductsError.message,
+      req: { body: req.body, params: req.params }
+    });
     const e = Error.http(error);
```

Comment:
can help me change this const e = Error.http(error) to const e = Error.http(getAllProductsError)